frappe_io/www/bench/index.md

```diff
@@ -1,36 +1,36 @@
 <section class='top-section'>
 	<h1>CLI to Manage Frappe Deployments</h1>
 	<p class='lead'>
-		Bench is a command line tool that helps you install, setup, manage multiple sites and apps based on Frappe Framework.
+		bench is a command line tool that helps you install, setup, manage multiple sites and apps based on Frappe Framework.
 	</p>
 </section>
 
 ### Create a new Frappe Environment
 
-You can run multiple instances of frappe + erpnext on one server by creating multiple benches
+You can have multiple sites running Frappe along with other Frappe based Apps (like ERPNext) on one bench and have different versions of Frappe (and Frappe apps) across multiple benches on the same server! Apart from this, most commands require to be run in the context of the respective bench directory.
 
 ```sh
-bench init my-bench
+bench init frappe-bench-dev
 ```
 
 ### Create and Manage Sites
 
-Bench will help you create new instances of ERPNext, called `sites` and also migrate and debug them.
+Before running any more commands be sure to change working directory to your bench! For this instance, the root directory is _frappe-bench-dev_. bench will help you create new sites. Each site can hold installations of various combinations of Frappe apps.
 
 ```sh
 # create a new site
-bench new-site mysite.local
+bench new-site new-site1.local
 
 # install erpnext in mysite.local
-bench --site mysite.local install-app erpnext
+bench --site new-site1.local install-app erpnext
 
 # backup the site
-bench --site mysite.local backup
+bench --site new-site1.local backup
 ```
 
 ### Manage Updates
 
-You can update all your instances to the latest release versions by running `bench update`. This will download the latest version of erpnext, update dependencies, run patches and also migrate the database schema.
+You can update all your instances to the latest release versions by running the update command. This will download the latest version all the apps on bench, update dependencies, run patches and also migrate the database schema. If executed outside a bench directory, it just updates the bench tool!
```

Comment:
```suggestion
You can update all your instances to the latest release versions by running the `update` command. This will download the latest version all the apps on bench, update dependencies, run patches, and also migrate the database schema. If executed outside a bench directory, it just updates the bench tool!
```